lang/node-mozilla-iot-gateway/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=node-mozilla-iot-gateway
+PKG_VERSION:=0.3.1
+PKG_RELEASE:=1
+
+PKG_REV:=0.3.1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/mozilla-iot/gateway.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_REV)
+PKG_SOURCE_VERSION=$(PKG_REV)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_REV).tar.gz
+PKG_MIRROR_HASH:=ba05bc3e93c36768244df922434e7132c2dae85a1ff9e3213beea087a4844d11
+
+PKG_BUILD_DEPENDS:=node/host openzwave
+PKG_NODE_VERSION:=8.10.0
+
+PKG_MAINTAINER:=Marko Ratkaj <marko.ratkaj@sartura.hr>
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_SUBDIR)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/node-mozilla-iot-gateway
+  DEPENDS:=+node-npm +libopenzwave +python +openssl-util
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Node.js
+  TITLE:=Mozilla IoT Gateway
+  URL:=https://iot.mozilla.org/gateway/
+endef
+
+define Package/node-mozilla-iot-gateway/description
+  Mozilla IoT Things Gateway
```

Comment:
that's not really a description, that's actually less information that the title, which includes that this is somehow node related, and it's also redundant with the T for things.

Reply:
"[Things Gateway](https://iot.mozilla.org/gateway/)" (or perhaps "Web of Things gateway") is the name of the software.

Reply:
Yes. name.  This is a description field.

Reply:
So "Things" would not be redundant with the T.

Reply:
@jefferyto i will update the description and title. Thank you for your comments. Trying to get to the bottom of the build issue on Travis first.

Reply:
@ratkaj Actually, the reason why the Travis build is failing is because it uses the ar71xx sdk to test build your package, but node isn't available for ar71xx (no FPU). I think the only way to "fix" this is to modify the Travis script to pick another platform sdk.

Reply:
@jefferyto Interesting.. I recently bumped node:
https://github.com/openwrt/packages/pull/5734
and several other node-* packages. How come Travis didn't fail there. Is there something I am missing?